process/block/metablock_test.go

```diff
@@ -839,7 +839,7 @@ func TestMetaProcessor_ApplyBodyToHeaderShouldWork(t *testing.T) {
 	mp, _ := blproc.NewMetaProcessor(arguments)
 
 	hdr := &block.MetaBlock{}
-	err := mp.ApplyBodyToHeader(hdr, nil)
+	_, err := mp.ApplyBodyToHeader(hdr, nil)
```

Comment:
Do not forget to remove comments on lines 643 and 656 in transactions.go (//what is this?) :D

Reply:
done.